Ktisis/Configuration.cs

```diff
@@ -44,6 +45,12 @@ public class Configuration : IPluginConfiguration {
 		public bool DrawLinesOnSkeleton { get; set; } = true;
 		public float SkeletonLineThickness { get; set; } = 2.0F;
 
+		// References
+		public Dictionary<int, ReferenceInfo> References { get; set; } = new();
```

Comment:
Couldn't this be a `List<ReferenceInfo>` instead? Don't see a need for a dictionary here.

Reply:
I've kept `Dictionary<int, ReferenceInfo>` because (AFAIK) you need a unique ID for the imgui window. Using a key to a dictionary prevents a collision more easily than a property on the object. Using the index of the item of the list would be no good either; if you had (A, A) and deleted the first A then it would probably look like nothing happened (as for imgui, only the second one disappeared). Let me know what you think.

Reply:
Uniqueness is a good point here and something I hadn't considered. I'm still not *quite* happy with this, but I wouldn't consider it a merge blocker.